roles/docker/tasks/centos.yml

```diff
@@ -1,20 +1,12 @@
 ---
   - name: ensure old versions of Docker are not present
     yum:
-      name: "{{ item }}"
+      name: ["docker", "docker-common", "docker-engine"]
       state: absent
-    with_items:
-      - docker
-      - docker-common
-      - docker-engine
```

Comment:
Just curious, are we still able to use `with_items` in 2.8?

Reply:
`with_items` in general is still allowed, but is [deprecated ](https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.7.html#using-a-loop-on-a-package-module-via-squash-actions) for use with package modules (apt, yum, pip).